src/commands/ShowDoc.ts

```diff
@@ -0,0 +1,74 @@
+import * as vscode from 'vscode'
+import * as path from "path";
+import * as parser from "../parser"
+import * as extension from "../extension"
+import * as fs from "fs"
+
+export const id = "showDoc"
```

Comment:
We use snake_case in this project to make contributors form godot project comfotable.